lib/ood_core/job/adapters/kubernetes/helper.rb

```diff
@@ -263,10 +263,11 @@ def submission_time(json_data)
   def pod_status_from_json(json_data)
     phase = json_data.dig(:status, :phase)
     conditions = json_data.dig(:status, :conditions)
+    container_statuses = json_data.dig(:status, :containerStatuses)
     unschedulable = conditions.to_a.any? { |c| c.dig(:reason) == "Unschedulable" }
+    ready = !container_statuses.to_a.empty? && container_statuses.to_a.all? { |s| s.dig(:ready) == true }
+    return "running" if ready
     state = case phase
```

Comment:
I don't think we have a rubocop here yet, but I believe it would like a newline after this return statement.

Reply:
Fixed.